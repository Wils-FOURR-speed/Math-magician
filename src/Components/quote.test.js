import React from 'react';
import renderer from 'react-test-renderer';
import ShowQuotes from './quote';

describe('ShowQuotes', () => {
  test('renders correctly', () => {
    const component = renderer.create(<ShowQuotes />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});