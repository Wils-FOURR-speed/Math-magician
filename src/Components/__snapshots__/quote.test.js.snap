// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowQuotes renders correctly 1`] = `
<p
  className="load"
>
  Loading...ðŸ¤”
</p>
`;
